collect_app/src/main/java/org/odk/collect/android/utilities/ImageConverter.java

```diff
@@ -39,9 +39,26 @@
     private ImageConverter() {
     }
 
+    /**
+     * Before proceed with scaling or rotating, make sure existing exif information is store/restored.
```

Comment:
Should be: stored/restored (d missing).

Reply:
Comment has been updated, please kindly review